Gopkg.toml

```diff
@@ -35,5 +35,5 @@
 
 [[constraint]]
   name = "github.com/pmorie/osb-broker-lib"
-  version = "0.0.5"
+  version = "0.0.7"
```

Comment:
@pmorie We really need to set up those e2e tests for this project, to make sure nothing breaks while bumping versions of the lib. ;)

Reply:
Yes we do -- too bad there's no 'kube CI as a service :('

Reply:
Now i'm getting some ideas, lol...